tools/accuracy_checker/configs/horizontal-text-detection-0001.yml

```diff
@@ -0,0 +1,38 @@
+models:
```

Comment:
@GalyaZalesskaya , we have important change connected with Accuracy Checker configs location, starting with openvino 2021.1 (it is target release for your model).
Please move this file to your model directory (where description stored) and rename it to `accuracy_check.yml`
see https://github.com/opencv/open_model_zoo/tree/develop/models/intel/text-detection-0004 as example.
in this place, you can create symlink (sometime we will need it for backward compatibility on infra side) or I can do it after model merge if we still will require this.